zimui/src/components/ErrorDisplay.vue

```diff
@@ -8,25 +8,96 @@ const errimage = ref(errimageData)
 </script>
 
 <template>
-  <div class="error-container">
-    <img :src="errimage" class="error-image" />
-    <p class="error-text">{{ main.errorMessage }}</p>
-    <p class="error-text">{{ main.errorDetails }}</p>
+  <div class="error-page">
+    <nav class="nav-header">
+      <div class="logo">Kolibri</div>
```

Comment:
"Kolibri" means nothing to the end-user, what the user knows is the Channel title or the ZIM title. Given we already have access to the channel (via channel.json file), please replace "Kolibri" with Kolibri channel title.